lib/hanami/view/helpers/escape_helper.rb

```diff
@@ -0,0 +1,301 @@
+# frozen_string_literal: true
+
+require "escape_utils"
+require "temple"
+require "uri"
+
+module Hanami
+  class View
+    module Helpers
+      # Escape helpers
+      #
+      # You can include this module inside your view and
+      # the view will have access all methods.
+      #
+      # By including <tt>Hanami::Helpers::EscapeHelper</tt> it will inject private
+      # methods as markup escape utilities.
+      #
+      # @since 0.1.0
```

Comment:
Thanks for clarifying this for me. Going from this, I updated all helpers to `@since 2.0.0` in https://github.com/hanami/view/pull/229/commits/76333b887ba5b8ae7d336abf02b9a47d0de8071f.